source/releasenotes/2024-02-21-php-8.3.3-8.2.16-8.1.27.md

```diff
@@ -0,0 +1,7 @@
+---
+title: PHP 8.1.27, 8.2.16, and 8.3.3 now available
+published_date: "2024-02-21"
```

Comment:
@IngridKwok @jazzsequence how do you want to handle date discrepancies? Is "February 21" meant to communicate the date on which these changes were made on the platform or the date of the publishing of the note?

I think ideally they'd be the same date, but we're a week late for that.

Reply:
@stevector @jazzsequence I recommend using the date the changes were made on the platform, that's Feb 21. My take is that our audience are more interested in knowing when the changes were made so we should prioritize that information.

Reply:
Feb 21 is the date the change was merged so that was what the date was intended to communicate. The PHP versions had all been available for weeks at least.

Reply:
Makes sense to me. I'll merge